### PR TITLE
feat(cloud-sql/postgres): update to V2 sample

### PR DESCRIPTION
**Pull request by jsimonweb**
- Move TCP and UNIX socket examples into separate files
- Update region tags
- Update DB_HOST env. variable to be INSTANCE_HOST
- Update DB_SOCKET_PATH env. variable to be INSTANCE_UNIX_SOCKET

### CODE REVIEW

File: cloud-sql/postgres/ConnectTcp.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        public static NpgsqlConnectionStringBuilder NewPostgreSqlTCPConnectionString()
+        {
+            // Equivalent connection string:
+            // "Uid=<DB_USER>;Pwd=<DB_PASS>;Host=<INSTANCE_HOST>;Database=<DB_NAME>;"
+            var connectionString = new NpgsqlConnectionStringBuilder()
+            {
+                // The Cloud SQL proxy provides encryption between the proxy and instance.
+                SslMode = SslMode.Disable,
+
+                // Remember - storing secrets in plain text is potentially unsafe. Consider using
```
[the remark this answers is not in the document]
Comment (enocom) [reply cloud-sql/postgres/ConnectTcp.cs:34]: https://github.com/GoogleCloudPlatform/golang-samples/blob/c0583a59e8b6f28c39437328996411a5988b27c5/cloudsql/postgres/database-sql/connect_tcp.go#L41-L44

File: cloud-sql/postgres/ConnectUnix.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+namespace CloudSql
+{
+    public class PostgreSqlUnix
+    {
+        public static NpgsqlConnectionStringBuilder NewPostgreSqlUnixSocketConnectionString()
+        {
+            // Equivalent connection string:
+            // "Server=<dbSocketDir>/<INSTANCE_CONNECTION_NAME>;Uid=<DB_USER>;Pwd=<DB_PASS>;Database=<DB_NAME>"
+            String dbSocketDir = Environment.GetEnvironmentVariable("INSTANCE_UNIX_SOCKET") ?? "/cloudsql";
```
Comment (enocom) [inline cloud-sql/postgres/ConnectUnix.cs:29]: Same here: Let's remove the `/cloudsql` alternative. `INSTANCE_UNIX_SOCKET` should be the full path.

**Review by enocom — CHANGES_REQUESTED** [review]

File: cloud-sql/postgres/ConnectTcp.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jsimonweb) [reply cloud-sql/postgres/ConnectTcp.cs]: Done.

File: cloud-sql/postgres/ConnectUnix.cs (hunk 2 above)
Comment (jsimonweb) [reply cloud-sql/postgres/ConnectUnix.cs]: Done.

**Review by enocom — APPROVED** [review]

**Review by amanda-tarafa — APPROVED** [review]
LGTM!

**Pull request merged**